src/Devices/EEPROM_TC.h

```diff
@@ -0,0 +1,86 @@
+#pragma once
+
+#include <Arduino.h>
+#include <EEPROM.h>
+
+class EEPROM_TC {
+public:
+  // class methods
+  static EEPROM_TC* instance();
+
+  // accessor methods
+  double getPH();
+  double getTemp();
+  double getTankID();
+  double getCorrectedTemp();
+  double getKP();
+  double getKI();
+  double getKD();
+  double getMac();  // Not used?
```

Comment:
The comment here is confusing me.

Its interesting to consider if the comment or getMac function should be included. We are storing things in EEPROM with specific address. If the function was removed, now we would have a hole in the data storage area which could be confusing. Although if we move all the functions below up, then old versions will have an issue.

Maybe for now, lets just create an issue to review if getMac() is being used.

@jgfoster Thoughts?

Reply:
Yes, let's create an issue "Investigate need for `getMac()`" and then replace the "Not used?" comment with "See <URL> for a discussion about this function"